### PR TITLE
Updated Code Addressing James comments and adding MDS plotting code

### PR DESCRIPTION
**Pull request by bmacias01**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,8 +11,8 @@ pandas = "^2.0.2"
 mypy = "^1.4.1"
 pydantic = "^1.10.9"
 pandas-stubs = "^2.0.2.230605"
```
Comment (jamesturk) [inline pyproject.toml:13]: Two issues here, first is you don't want to remove networkx/distinctipy, second is `sklearn` is https://pypi.org/project/scikit-learn/ and should be installed as `scikit-learn`.

**Review by jamesturk — COMMENTED** [review]
a quick review looks good to me with the exception of a pyproject.toml issue, someone might want to give the actual code a second pass if they have time

File: pyproject.toml (hunk 1 above)
Comment (bmacias01) [reply pyproject.toml:13]: ok should I readd those then do a another pull request?

Comment (jgibson517) [reply pyproject.toml:14]: I took care of it Brenda! it was an easy fix

**Pull request closed without merging**